### PR TITLE
fix govet error

### PR DESCRIPTION
**Pull request by pei0804**
**Describe the PR**
fix for govet error and add go vet check.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -55,6 +55,7 @@ lint:
 	fi
 	
 	for PKG in $(PACKAGES); do golint -set_exit_status $$PKG || exit 1; done;
+	go vet
```
Comment (easonlin404) [inline Makefile]: maybe can add new .PHONY:vet like:

``` sh
.PHONY: vet
vet:
 $(GO) vet $(PACKAGES)
```

Comment (easonlin404) [reply Makefile]: and add to https://github.com/swaggo/swag/blob/889705a8db7763121543be8c314d836ebf040d42/.travis.yml#L19-L23

Comment (pei0804) [reply Makefile]: ok

**Comment by pei0804** [comment]
@easonlin404 review plz

**Pull request merged**